### DISCUSSION TITLE
Exporting to a file?

### DISCUSSION THREAD

**Discussion by fpolli** [Beakerboy/OSMBuilding#70] [Q&A] [answered]
What about saving a building to gltf? Can it do that?

**Comment by Beakerboy** [comment, marked as answer]
It cannot. This sounds like an idea for a separate project though. Feel free to borrow any of my code or guidance on algorithms if you decide to make a tool that does this.

**Reply by fpolli** [reply]
Thanks! I wish I were qualified to make a tool like that.

Sorry, coming back to this. You are using three.js? There is a gltf export plugin for three.js

https://threejs.org/docs/#examples/en/exporters/GLTFExporter

I might decide to play around with it, maybe it will work here.

**Reply by Beakerboy** [reply]
Yes, I’m using three.js. 

> I wish I were qualified to make a tool like that.

don’t let that stop you. I’m “unqualified” for all of this too. One day I said I wanted to figure out how to render 3D buildings on the fly, so I’ve been teaching myself JavaScript, three.js, and CI/CD with JavaScript by doing this project. “Now” is always the best time to learn something new.
